### ISSUE TITLE
Regression in example child plugins

### ISSUE THREAD

**Issue by amitaibu** [RESTful-Drupal/restful#210]
![local_d7_dev_api-3](https://cloud.githubusercontent.com/assets/125707/4780990/65deaed8-5c83-11e4-828c-f154d6616f00.jpg)

Also, when I click on `api/v1.0/anonymous-user` I expected to get only nodes with anonymous roles.

![local_d7_dev_api_v1 0_anonymous-user](https://cloud.githubusercontent.com/assets/125707/4780993/88f864cc-5c83-11e4-946d-281d4a4ccfaa.jpg)

**Comment by e0ipso** [comment]
True, we should work adapt the variations in the children generation function.
